tests/test_array_type.py

```diff
@@ -0,0 +1,388 @@
+from typing import Iterable
+
+import numpy as np
+import pytest
+
+import pixeltable as pxt
+import pixeltable.exceptions as excs
+from pixeltable import type_system as ts
+from pixeltable.type_system import ArrayType, ColumnType, IntType
+
+from .utils import reload_catalog, validate_update_status
+
+
+class TestArrayType:
```

Comment:
I didn't read the test code super carefully so it's possible this is already happening, but let's make sure we test the output of `str(Array[...])` for each of the supported array dtypes.